MediaAdminBundle/Resources/views/BackOffice/Underscore/mediaModalView._tpl.twig

```diff
@@ -1,9 +1,18 @@
-<div class="modal fade mediaModalContainer divMessageBox" tabindex="-1" role="dialog" aria-labelledby="" aria-hidden="true" data-backdrop="static" data-input="<%= input %>">
+<div
+    class="modal fade mediaModalContainer
+    divMessageBox" tabindex="-1" role="dialog"
```

Comment:
Are you sure that a new line in the middle of an HTML attribute value is working well ?